keyboards/dyz/synthesis60/info.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
This file needs to be redone without the rotation:
![image](https://user-images.githubusercontent.com/4781841/149426914-64ec0b8e-af99-435a-bb42-c0385f56f8e0.png)